### PR TITLE
Mjml carousel

### PR DESCRIPTION
**Pull request by kmcb777**

### CODE REVIEW

File: packages/mjml-carousel/src/Carousel.js [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+    if (!length) return ''
+
+    const carouselCss = `
+    .mj-carousel {
+      -webkit-user-select: none;
+      -moz-user-select: none;
+      user-select: none;
+    }
+
+    .mj-carousel-icons-cell {
```
Comment (iRyusa) [inline packages/mjml-carousel/src/Carousel.js]: You don't need to prefix it with the carouselID too ?

Comment (kmcb777) [reply packages/mjml-carousel/src/Carousel.js]: No, this class is not involved with the checked/hover behaviors so it's fine

Comment (iRyusa) [reply packages/mjml-carousel/src/Carousel.js]: Ok but cell width is related to current carousel no (cf the getAttribute right after) ?

Comment (kmcb777) [reply packages/mjml-carousel/src/Carousel.js]: That's actually very true, i'll fix that

**Pull request merged**